core/src/test/java/org/apache/calcite/sql/test/SqlAdvisorTest.java

```diff
@@ -175,6 +175,7 @@
           "KEYWORD(LAG)",
           "KEYWORD(LAST_VALUE)",
           "KEYWORD(LEAD)",
+          "KEYWORD(LISTAGG)",
```

Comment:
why? If this is keyword, I can't use it as column alias or table name. It doesn't make sense.